cg/constants/pacbio.py

```diff
@@ -17,6 +17,15 @@ class PacBioDirsAndFiles:
     RUN_IS_VALID: str = "is_valid"
 
 
+class MetricsFileFields:
+    ATTRIBUTES: str = "attributes"
+    COLUMNS: str = "columns"
+    ID: str = "id"
+    TABLES: str = "tables"
+    VALUE_ONE: str = "value"
+    VALUES_MANY: str = "values"
```

Comment:
I think breaking the established pattern is more troublesome than potential typos. I would expect `VALUE_ONE = "value_one"` given how the rest is written.

```suggestion
    VALUE: str = "value"
    VALUES: str = "values"
```

Reply:
I was unsure of this one, thanks for the feedback, I will change it